src/ctia/bulk/schemas.clj

```diff
@@ -28,20 +28,23 @@
   "Error related to one entity of the bulk"
   {:error s/Any})
 
-; TODO def => defn
-(s/defschema Bulk
-  (entities-bulk-schema (entities/all-entities) :schema))
+(s/defn get-entities :- [s/Any]
+  "Returns list of enabled entities"
+  [{{:keys [enabled?]} :FeaturesService} :- APIHandlerServices]
```

Comment:
It might help to use a more permissive schema here to get the tests to pass with schema enabled.

eg., `HTTPShowServices` exists for similar reasons https://github.com/threatgrid/ctia/blob/e6b45249024c73093ae7692e9b7ff1cbef207f7d/src/ctia/schemas/core.clj#L46

Reply:
I'm not exactly sure what you're saying. Why does it have to be more open? So we don't have to do something like this? https://github.com/threatgrid/ctia/blob/6382bd7753dc4e3aceceb451ad94e05a52ab1985/test/ctia/bulk/core_test.clj#L36

Reply:
Yes, it's handy for testing with a subset of services in tests.

Reply:
Yes, we should do everything we can to avoid `(s/set-fn-validation! false)`, especially if this PR introduces the schemas that fail in the first place.